lib/api/core/auth/passwordManager.js

```diff
@@ -0,0 +1,21 @@
+var crypto = require('crypto');
+
+module.exports = function PassportManager () {
+
+  this.secret = 'jbsB8r69PFP39KdLtjVr25Z22';
```

Comment:
it's just a first step to have something usable.
We need to improve the passwordManager and of course let the secret, the algo be configurable, + probably add other features.
But I prefer working step by step and delay this to a later PR ;)